internal/file/tar_file_traversal.go

```diff
@@ -2,7 +2,7 @@ package file
 
 import (
 	"fmt"
-	"io/ioutil"
```

Comment:
also a lint fix that was not changed with the latest update `ioutil` depreciation